strictdoc/backend/sdoc/grammar/grammar.py

```diff
@@ -94,7 +112,16 @@
 ;
 
 SectionOrRequirement[noskipws]:
-  '\n' (Section | Requirement | CompositeRequirement)
+  '\n' (Section | Requirement | CompositeRequirement | IncludeFromFile)
+;
+
+IncludeFromFile[noskipws]:
+  '[INCLUDE_SECTION_FROM_FILE]' '\n'
+  ('UID: ' uid = /.+$/ '\n')?
```

Comment:
Is there a use case for maintaining these three optional fields: `UID`, `LEVEL` and `TITLE`?